### PR TITLE
util to figure out cdap version of metric's hbase table

### PR DESCRIPTION
**Pull request by abaranau**
https://issues.cask.co/browse/CDAP-1522

### CODE REVIEW

**Comment by abaranau** [comment]
http://builds.cask.co/browse/CDAP-DUT915-1

**Comment by abaranau** [comment]
GREEN

File: cdap-data-fabric/src/main/java/co/cask/cdap/data2/dataset2/lib/table/hbase/HBaseTableAdmin.java [hunk 1]
```diff
@@ -81,6 +81,7 @@ public void create() throws IOException {
 
 
     final HTableDescriptor tableDescriptor = new HTableDescriptor(name);
+    setVersion(tableDescriptor);
```
Comment (abaranau) [inline cdap-data-fabric/src/main/java/co/cask/cdap/data2/dataset2/lib/table/hbase/HBaseTableAdmin.java]: yes, we were not adding cdap version property to tables up until now.. we only did on table upgrade

**Comment by ghelmling** [comment]
lgtm

**Pull request merged**